spec/models/book_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+require 'factories'
+
+RSpec.describe Book, type: :model do
+  subject(:book) { build(:book) }
```

Comment:
`create(:book)` I know it's not a very huge change but it's preferable to test the model once it's created in the DB